### DISCUSSION TITLE
Survey on Sympa, 2024

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1858] [Ideas]
The first survey about Sympa was conducted in 2006, nine years after Sympa's first release.

The second survey was conducted in 2015, another nine years later.

Nine more years later, this year is 2024, so may we have to conduct a survey...?

If we do, I propose that we would like to start distributing the survey in the summer and finish collecting responses by the end of this year.

Past survey results can be found here: https://assets.sympa.community/survey/

Also, please suggest any new questions that we should add this time (e.g., challenges in dealing with ARC).

**Comment by ldidry** [comment]
Suggestion of questions:

- Do you have delivery troubles with some mail providers, that are sympa-related? (I know that I had to patch Sympa to let Orange accepts some mails)
- Would you like a page on https://sympa.community with the most common mail providers problems and the solutions found by the community? (Like: the Orange problems (plural is not a mistake), ATT that asks to be contacted to be able to send them mails, Hotmail being a pain in the ass…)
- Would you like to contribute to such a page?
